benchmark/Streamly/Benchmark/Data/Stream/Eliminate.hs

```diff
@@ -453,23 +488,25 @@ o_1_space_elimination_folds value =
         , benchIOSink value "maximum" maximum
         , benchIOSink value "minimumBy" minimumBy
         , benchIOSink value "minimum" minimum
-
+#ifdef USE_PRELUDE
         , bench "the" $ nfIO $ randomRIO (1,1) >>= the . repeat value
+#endif
         , benchIOSink value "find" (find value)
         , benchIOSink value "findM" (findM value)
-        -- , benchIOSink value "lookupFirst" (lookup 1)
+        , benchIOSink value "lookupFirst" (lookup 1)
```

Comment:
Uncommenting this will create problems with reporting of benchmarking times, beacuse all times now will come in nanoseconds, as this benchmark is too small.